pom.xml

```diff
@@ -88,6 +88,7 @@
         <jmh.version>1.19</jmh.version>
         <se.eris.notnull.instrument>false</se.eris.notnull.instrument><!-- off by default -->
         <slf4j.version>1.7.25</slf4j.version>
+        <com.uber.jaeger.version>0.23.0</com.uber.jaeger.version>
```

Comment:
this indirection is redundant - use the version number where it's needed (1 place).

Reply:
resolved